docs/lib/analytics/fragments/ios/getting-started/40_record.md

```diff
@@ -0,0 +1,14 @@
+To record an event,  specify your event using `BasicAnalyticsEvent` and call `Amplify.Analytics.record()`
+
+```swift
+func recordEvents() {
+    let properties: [String: AnalyticsPropertyValue] = [
```

Comment:
there is a typealias for this
`public typealias AnalyticsProperties = [String: AnalyticsPropertyValue]`

`let properties: AnalyticsProperties = [...`

Reply:
changed